packages/exporter-trace-otlp-http/src/util.ts

```diff
@@ -39,3 +42,44 @@ export function appendResourcePathToUrlIfNotPresent(url: string, path: string):
 
   return url + path;
 }
+
+/**
+ * Configure exporter trace timeout value from passed in value or environment variables
+ * @param timeoutMillis
+ * @returns timeout value in milliseconds
+ */
+
+export function configureExporterTimeout(timeoutMillis: number | undefined): number {
+  if (typeof timeoutMillis === 'number') {
+    if (timeoutMillis <= 0) {
+      // OTLP exporter configured timeout - using default value of 10000ms
+      return invalidTimeout(timeoutMillis, DEFAULT_TRACE_TIMEOUT);
+    }
+    return timeoutMillis;
+  } else {
+    return getExporterTimeoutFromEnv();
+  }
+}
+
+function getExporterTimeoutFromEnv(): number {
+  const definedTimeout =
+    Number(getEnv().OTEL_EXPORTER_OTLP_TRACES_TIMEOUT ||
+    getEnv().OTEL_EXPORTER_OTLP_TIMEOUT);
+
+  if (definedTimeout) {
```

Comment:
This will be false if the value is zero (0), so the next line <= 0 is really only < 0